packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,278 @@
+# Terra Application Layout
+
+## Prerequisites
+
+- Like all Terra components, the ApplicationLayout requires the presence of a `Base` component (provided by `terra-base`) in its parent hierarchy.
+- Additionally, the ApplicationLayout requires the presence of any `Router` component (provided by `react-router-dom`) in its parent hierarchy.
```

Comment:
Part of me thinks we should add a code sample here:

```js
import React from 'react';
import { BrowserRouter } from 'react-router-dom'
import Base from 'terra-base';
import ApplicationLayout from 'terra-application-layout';

<BrowserRouter> // This can be swapped with MemoryRouter or HashRouter per your use-case
  <Base locale="en-US">
    <ApplicationLayout
        nameConfig={nameConfig}
        utilityConfig={utilityConfig}
        routingConfig={routingConfig}
        navigationItems={navigationItems}
        extensions={<ApplicationExtensions />}
        indexPath={indexPath}
      />
  </Base>
</BrowserRouter>
```